docs/microphone.rst

```diff
@@ -0,0 +1,83 @@
+Microphone **V2**
+*****************
+
+.. py::module:: microbit
+
+This object lets you access the on-board microphone available on the **V2**
+micro:bit version. It can be used to respond to sound. The microphone input
+is located on the front of the board alongside a microphone activity LED,
+which is lit when the microphone is in use.
+
+.. image:: microphone.png
+    :width: 300px
+    :align: center
+    :height: 240px
+    :alt: micro:bit with microphone LED on
+
+Sound events
+============
+The microphone can respond to a pre-defined set of sound events that are
+based on the amplitude and wavelength of the sound.
+
+These sound events are identified by strings passed as method arguments and
+return values. These strings are also stored as `MicrobitMicrophone` class
+variables as a handy way to identify what events are available.
+
+Classes
+=======
+.. py:class:: MicrobitMicrophone
+
+    .. note::
+        The ``MicrobitMicroPhone`` class documented here can be accessed via 
+        the microbit module as ``microbit.microphone``. It cannot be
+        initialised on it's own.
+
+    Represents the microphone.
+
+.. py:class:: SoundEvent  
+
+    Value to represent loud sound events, like clapping or shouting
+    ``SoundEvent.LOUD`` = ``loud``.
```

Comment:
These are internally represented as strings, but to the user they are instances of the SoundEvent class.

Reply:
@microbit-mark This still needs to be updated.
Related to: https://github.com/bbcmicrobit/micropython/pull/692#discussion_r530518765

Reply:
Already fixed.